api/v1alpha1/instancegroup_types.go

```diff
@@ -487,6 +495,12 @@ func (c *EKSConfiguration) Validate() error {
 			return err
 		}
 	}
+
+	for i, v := range c.LicenseSpecifications {
+		if !common.StringEmpty(v) && !strings.HasPrefix(v, awsprovider.ARNPrefix) {
```

Comment:
Should this be || instead?

Reply:
its based upon the logic above: https://github.com/keikoproj/instance-manager/pull/199/files/03b5e05aa1abae87f64cc14bcc05154451ab9cf9#diff-9f09697a5f636c8e5c9996db1dac259666efd6f1631577aa6f4bb814acd685f8R447

we could flip the logical statement to:

`if common.StringEmpty(v) || strings.HasPrefix(v, awsprovider.ARNPrefix) { //pass }` 

The way I would express it in English is; the LicenseSpecifications array must only contain empty strings, or valid ARNs.

Reply:
What does it mean if it contains an empty string?
Isn't an empty string considered invalid? it would mean someone provided it as:
```yaml
configuration:
  licenseSpecifications:
  - ""
```
In that case, shouldnt it be: `the LicenseSpecifications array must only contain valid ARNs.`

Reply:
yeah that's fair.